exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -66,9 +74,44 @@ impl JavaServiceRuntime {
         }
     }
 
-    /// Returns internal service proxy.
-    pub fn service_proxy(&self) -> ServiceProxy {
-        self.service_proxy.clone()
+    /// Loads an artifact from the specified location involving verification of the artifact.
+    /// Returns an unique service artifact identifier that must be specified in subsequent
+    /// operations with it.
+    pub fn load_artifact(&self, artifact_uri: &str) -> String {
+        unwrap_jni(self.executor.with_attached(|env| {
+            let artifact_uri: JObject = unwrap_jni(env.new_string(artifact_uri)).into();
+            let artifact_id: JObject = env
+                .call_method(
+                    self.service_runtime.as_obj(),
+                    "loadArtifact",
+                    LOAD_ARTIFACT_SIGNATURE,
+                    &[artifact_uri.into()],
+                )?
+                .l()?;
+            let result: String = JavaStr::from_env(env, artifact_id.into())?.into();
+            Ok(result)
+        }))
+    }
+
+    /// Creates a new service instance of the given type.
```

Comment:
What given type?

Reply:
For the moment I just copied it from the corresponding Java methods. Going to improve.